site/styles/_variables.scss

```diff
@@ -240,6 +245,9 @@ $signup-border-color-hover: transparent !default;
 $signup-background: $button-background !default;
 $signup-background-hover: $button-background-hover !default;
 
+// black image is available veriplay-col-black.png
+$veriplay-logo-path: '/images/common/veriplay-col-white.png';
```

Comment:
Sass variables aren't supported by shared-template CSS... This will need to be `--veriplay-logo-path` in the `:root` block above instead.

Reply:
Yeah I have tried a number of things here

![image](https://user-images.githubusercontent.com/99003/204975609-994d495b-de20-4142-953a-40d3fb2e5a91.png)

Not sure if this is a sass limitation?

Reply:
Did you try this: https://stackoverflow.com/a/42331003

Reply:
Thanks for this!